book/Kibble; Kibbles' Crafting Guide.json

```diff
@@ -0,0 +1,861 @@
+{
+	"_meta": {
+		"sources": [
+			{
+				"json": "kibblescrafting",
+				"abbreviation": "K:CG",
+				"full": "Kibbles' Crafting Guide",
+				"authors": [
+					"Jeff Gomez",
+					"EN5ider"
+				],
+				"convertedBy": [
+					"cadowtin"
+				],
+				"version": "1.0",
+				"url": "https://www.patreon.com/posts/crafting-1-0-6-58649169",
+				"targetSchema": "1.0.0"
+			}
+		],
+		"dateAdded": 1549715846,
+		"dateLastModified": 1549715846
+	},
+	"book": [
+		{
+			"name": "Kibbles' Crafting Guide",
+			"id": "K:CG",
+			"source": "kibblescrafting",
+			"published": "2021-11-13",
+			"group": "homebrew",
+			"contents": [
+				{
+					"name": "Kibbles' Crafting Guide"
+				}
+			]
+		}
+	],
+	"bookData": [
+		{
+			"id": "K:CG",
+			"source": "kibblescrafting",
+			"data": [
+
+			]
+		}
+	],
+	"table": [
+
+	],
+	"item": [
+		{
+			"name": "Firewood",
+			"source": "K:CG",
```

Comment:
these sources all need to be `"kibblescrafting"` to match the source `"json"` declared in your `"_meta"`